play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -457,76 +461,169 @@ Proficient: Explosives
+		once per separate die use and if you reroll, choose which roll to keep.
+	Luck = 9		
+		You can choose to reroll a die, three times per long rest. You can only
+		reroll once per separate die use and if you reroll, choose which roll to keep.
+	Luck = 10		
+		You can choose to reroll a die, four times per long rest. You can reroll
+		as many times as you want (which uses up your rerolls-per-rest) and if you 
+		reroll, choose which of the rolls to keep.
+	Luck = 12	
+		You can choose to reroll a die with +10% or -10% bonus, four times per 
+		long rest. You can reroll as many times as you want (which uses up your 
+		rerolls-per-rest) and if you reroll, choose which of the rolls to keep.
+	Luck = 15	
+		You can choose to reroll a die with +20% or -20% bonus, four times per 
+		long rest. You can reroll as many times as you want (which uses up your
+		rerolls-per-rest) and if you reroll, choose which of the rolls to keep.
+		
+Positional Mastermind: 
+Prerequisites: Mutually Exclusive with Will to Survive; Charisma 9
+	You have incredible understanding of how beings in your universe perceive and react
+to their environments, allowing you to feint and position yourself in cover better. 
+If you have 9 Charisma, any living creature targeting you gets a +1 miss chance penalty
+when you have any kind of cover. If you have 12 Charisma, anything targeting you gets a 
++1 miss chance penalty when you have any kind of cover.
+
+Will to Survive: 
+Prerequisites: Mutually Exclusive with Positional Mastermind; Charisma 6
+	You are convinced that you will survive no matter the hardship. *does not purge debuffs unless 
+you have 14 Charisma*	
+	Charisma = 6
+			Once per encounter, once per turn if you aren't knocked out, as an immediate
+			reaction to being at or below half health, roll 1d10. If  <= 5, heal one wound
+			up for up to +15 hitpoints. If >5 heal one wound for up to +20 hitpoints.	
+	Charisma = 8	
+			Twice per encounter, once per turn if you aren't knocked out, as an immediate 
+			reaction to being at or below half health, roll 1d10. If  <= 5, heal one wound
+			up for up to +15 hitpoints. If >5 heal one wound for up to +20 hitpoints.	
+	Charisma = 10
+			Twice per encounter, once per turn if you aren't knocked out, as an immediate 
+			reaction to being at or below half health, roll 1d10. If  <= 5, heal one wound
+			up for up to +20 hitpoints. If >5 heal one wound for up to +30 hitpoints.	
+	Charisma = 12
+			Twice per encounter, once per turn if you aren't knocked out, as an immediate 
+			reaction to being at or below half health, roll 1d10. If  <= 5, heal one wound
+			up for up to +25 hitpoints. If >5 heal one wound for up to +40 hitpoints.
+	Charisma = 14
+			Twice per encounter, once per turn if you aren't knocked out, as an immediate 
+			reaction to being at or below half health, roll 1d10. If  <= 5, heal one wound
+			up for up to +25 hitpoints. If >5 heal one wound for up to +40 hitpoints. In 
+			addition, purge 1 debuff.

```

Comment:
Mildly concerned by 'purge 1 debuff' with no qualifiers or restrictions. Not that I'm going to try arguing that permanent blindness is a debuff, but there are horror stories in Pathfinder of a maneuver known as Iron Heart Surge.

Reply:
Lolol Iron heart surge is pretty funny. Good catch, thanks!